docs/blog/how-to-use-secrets.md

```diff
@@ -0,0 +1,112 @@
+# Handling secrets with phabalicious
+
+## What is a secret?
+
+A **secret** is a password, a token, or any other information which should not be publicly visible or available. Secrets **must not** be stored in a fabfile, or anywhere else, as they are most likely deployed to other systems and might pose a security-risk.
+
+Secrets should be injected on runtime via e.g. environment variables or other mechanisms to reduce the attack surface.
+
+## Support for secrets in phabalicious
+
+Phabalicious makes it easy to declare secrets and has various ways to obtain the actual values:
+
+-   from environment variables or an .env-file
+-   via command-line-options
+-   from 1password (cli/connect)
+-   as a last resort by asking the user
+
+## Example
+
+### Step 1: Preparation
+
+Let's start with a very simple fabfile:
+
+```yaml
+name: Secrets in phabalicious
+
+needs:
+  - script
+  - local
+
+scripts:
+  demo:
+    # An example on how to consume a secret
+    - echo "host.fooSecret is %host.fooSecret%"
+
+hosts:
+  local:
+    fooSecret: bar
+```
+
+* Save the file as `fabfile.yaml` 
+* Run the command `phab -clocal script test`
```

Comment:
I think we should use `phab -clocal script demo` (demo instead of test), shouldn't we?

Reply:
true